### ISSUE TITLE
external link will not allow refresh of app. cannot refresh

### ISSUE THREAD

**Issue by dockery** [phonegap/phonegap-app-desktop#385]
adding in a simple <a href="...">click here</a> tag takes me to an external page. This page cannot return to app. App will not reset or turn off. I have had to remove and reload app 5 times. failure.

Tried using suggestions from other sites, but continued failure with ios device, even when using target="_blank"

Distressing.

**Comment by hermwong** [comment]
Are you trying this from the `PhoneGap Developer App`?

**Comment by dockery** [comment]
Yes. Building this for an adobe class in app development. I eventually just canned the project  :(
External links have numerous issues across many, many forums and domains. I chose instead to kill any exterior links. This is simply an issue with too many people requesting solutions and no answers. I canned to project in lieu of just showing a picture.

Thanks anyway. I'll look for an update in phonegap after a few months. This thread can be shuttered.
